lib/osf-components/addon/components/registries/page-renderer/template.hbs

```diff
@@ -0,0 +1,11 @@
+{{#each this.pageManager.schemaBlockGroups as |group|}}
+    <Registries::SchemaBlockGroupRenderer
+        @schemaBlockGroup={{group}}
+        @renderStrategy={{
+            component
+            'registries/schema-block-renderer/editable/mapper'
+            changeset=this.pageManager.changeset
+            node=@node
```

Comment:
`node` isn't mentioned in `PageRenderer`'s backing class -- should be listed as required/optional

(yearning for the day we have type checking that can catch this sort of thing)

Reply:
Will add node as a required parameter on `page-renderer`